### PR TITLE
Added Bearsampp

### PR DESCRIPTION
**Pull request by N6REJ**

### CODE REVIEW

**Review by QUGL-98-06-15 — COMMENTED** [review]
HTTPS

**Comment by N6REJ** [comment]
> HTTPS

it has https is it not supposed to?

**Comment by klaude** [comment]
That was a spam account. I've blocked them, but can't remove their comment because they made themselves a reviewer of this pull request. 

HTTPS is great! Thanks for the PR and for including Contributor Covenant on your website. I'll defer to @CoralineAda to make sure all the i's are dotted and t's are crossed before merging this PR.

**Review by CoralineAda — APPROVED** [review]

**Pull request merged**
